### PR TITLE
edits and updates

### PR DESCRIPTION
**Pull request by lacava**

### CODE REVIEW

File: build/build.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ if [ "${BUILD_PDF:-}" != "false" ] && [ -n "$DOCKER_EXISTS" ]; then
   fi
   if [ -d output/images ]; then rm -rf output/images; fi  # if images is a directory, remove it
   cp -R -L content/images output/
-  docker run \
+  sudo docker run \
```
Comment (trangdata) [inline build/build.sh]: I don't think this needs to be changed. The manuscript builds on CI without `sudo`.

File: content/01.summary.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ## Summary
 
 PMLB (Penn Machine Learning Benchmark) is an open source data repository containing a curated collection of datasets for evaluating and comparing machine learning (ML) algorithms.
-Compiled from a broad range of existing ML benchmark collections, PMLB unifies over 150 publicly available datasets from diverse sources such as Kaggle and OpenML, enabling systematic assessment of different ML methods.
+Compiled from a broad range of existing ML benchmark collections, PMLB unifies over 300 publicly available datasets from diverse sources such as Kaggle and OpenML, enabling systematic assessment of different ML methods.
```
Comment (trangdata) [inline content/01.summary.md]:
```suggestion
Compiled from a broad range of existing ML benchmark collections, PMLB unifies nearly 300 publicly available datasets from diverse sources such as Kaggle and OpenML, enabling systematic assessment of different ML methods.
```

File: content/01.summary.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
 These datasets cover a range of applications, from binary/multi-class classification to regression problems with combinations of categorical and continuous features.
-PMLB has both a Python interface (`pmlb`) and an R interface (`pmlbr`), both with detailed documentation that allow the user to access datasets using a simple `fetch_data` function.
-PMLB also offers a comprehensive description of each dataset with pandas profiling and advanced functions to explore the dataset space such as `nearest_datasets` and `filter_datasets`, which allow for smoother user experience and handling of data.
+PMLB has both a Python interface (`pmlb`) and an R interface (`pmlbr`), both with detailed documentation that allows the user to access cleaned and formatted datasets using a single function call (`fetch_data`).
+PMLB also provides a comprehensive description of each dataset and advanced functions to explore the dataset space such as `nearest_datasets` and `filter_datasets`, which allow for smoother user experience and handling of data.
+The resource is designed to facilitate open source contributions, in the form of datasets as well as improvements to curation.
```
Comment (trangdata) [inline content/01.summary.md]:
```suggestion
The resource is designed to facilitate open source contributions in the form of datasets as well as improvements to curation.
```

File: content/03.methods.md [hunk 4]
```diff
@@ -32,10 +32,13 @@ However, because the original source code was released under the [GNU General Pu
...
-This functionality would be helpful for users who would like to find PMLB datasets with similar characteristics to their own to make inference on their dataset, e.g., where to start the hyperparameter search.
+The core function of PMLB, `fetch_data()`, retains previous behavior by default, which excludes all rows with missing data (`dropna=True`). 
+However, if the user chooses to treat the missing values differently, they can use `fetch_data()` with the option `dropna=False` to obtain the original dataset and apply their own removal or imputation method.
+
+The new version of PMLB also allows the user to select datasets most similar to one of their own using the `nearest_datasets()` function. 
+Here, similarity between datasets is configurable to any number of metadata characteristics defined for the datasets. 
+By default, the nearest datasets in terms of number of samples and features is returned. 
+This functionality should be helpful for users who would like to find PMLB datasets with similar characteristics to their own in order to test or optimize methods (e.g. hyperparameter tuning) for their desired problem without the risk of over-fitting to their dataset.
```
Comment (trangdata) [inline content/03.methods.md]:
```suggestion
This functionality is helpful for users who look for PMLB datasets with similar characteristics to their own in order to test or optimize methods (e.g. hyperparameter tuning) for their desired problem without the risk of over-fitting to their dataset. 
```

File: content/03.methods.md [hunk 5]
```diff
@@ -32,10 +32,13 @@ However, because the original source code was released under the [GNU General Pu
...
-Defining the neighborhood to be the datasets' metadata/characteristics space, we also enabled the option to select the nearest PMLB datasets given a data frame.
-This functionality would be helpful for users who would like to find PMLB datasets with similar characteristics to their own to make inference on their dataset, e.g., where to start the hyperparameter search.
+The core function of PMLB, `fetch_data()`, retains previous behavior by default, which excludes all rows with missing data (`dropna=True`). 
+However, if the user chooses to treat the missing values differently, they can use `fetch_data()` with the option `dropna=False` to obtain the original dataset and apply their own removal or imputation method.
+
+The new version of PMLB also allows the user to select datasets most similar to one of their own using the `nearest_datasets()` function. 
+Here, similarity between datasets is configurable to any number of metadata characteristics defined for the datasets. 
+By default, the nearest datasets in terms of number of samples and features is returned.
```
Comment (trangdata) [inline content/03.methods.md]:
```suggestion
By default, the names of the nearest datasets are returned based on the number of samples and features.
```

File: content/03.methods.md [hunk 6]
```diff
@@ -32,10 +32,13 @@ However, because the original source code was released under the [GNU General Pu
...
-However, if the user chooses to treat the missing values differently, they can use fetch_data() with the option `dropna=False` to obtain the original dataset and apply their own removal or imputation method.
-Defining the neighborhood to be the datasets' metadata/characteristics space, we also enabled the option to select the nearest PMLB datasets given a data frame.
-This functionality would be helpful for users who would like to find PMLB datasets with similar characteristics to their own to make inference on their dataset, e.g., where to start the hyperparameter search.
+The core function of PMLB, `fetch_data()`, retains previous behavior by default, which excludes all rows with missing data (`dropna=True`). 
+However, if the user chooses to treat the missing values differently, they can use `fetch_data()` with the option `dropna=False` to obtain the original dataset and apply their own removal or imputation method.
+
+The new version of PMLB also allows the user to select datasets most similar to one of their own using the `nearest_datasets()` function. 
+Here, similarity between datasets is configurable to any number of metadata characteristics defined for the datasets.
```
Comment (trangdata) [inline content/03.methods.md]:
```suggestion
Here, the similarity between datasets is configurable to any number of metadata characteristics.
```

**Review by trangdata — COMMENTED** [review]
Great edits and suggestions @lacava. I only had a few minor suggestions.

File: build/build.sh (hunk 1 above)
Comment (lacava) [reply build/build.sh]: my mistake, fixed in 0923604

File: content/03.methods.md (hunk 4 above)
Comment (lacava) [reply content/03.methods.md]: how about "wish to find"

File: content/01.summary.md (hunk 2 above)
Comment (lacava) [reply content/01.summary.md]: check de0499f

File: content/03.methods.md (hunk 4 above)
Comment (trangdata) [reply content/03.methods.md]: Sounds great! Please feel free to merge when you're ready!

**Pull request merged**